big_tests/tests/inbox_SUITE.erl

```diff
@@ -78,7 +78,9 @@ groups() ->
        reset_unread_counter_and_show_only_unread,
        check_total_unread_count_and_active_conv_count,
        check_total_unread_count_when_there_are_no_active_conversations,
-       total_unread_count_and_active_convs_are_zero_at_no_activity
+       total_unread_count_and_active_convs_are_zero_at_no_activity,
+       inbox_result_cannot_exceed_configured_limit,
+       should_apply_lower_limit_when_both_provided
```

Comment:
Careful, this group is ran in parallel for all except odbc (see `inbox_helper:maybe_run_in_parallel/1`), so the `init_per_testcase` for it will mess the config. You probably need to add a separate group with its own `init_per_group` or so 😕